### ISSUE TITLE
Training code released？

### ISSUE THREAD

**Issue by DongxiaLiu15** [bytedance/lynx#8]
I would like to know if you have any idea to release the training code. I would like to know how the loss function is designed.

**Comment by tiancheng-zhi** [comment]
Sorry there is no plan to release training code. However, the loss function is same as Wan2.1 itself, a standard flow matching loss.

**Comment by nano-vision** [comment]
@tiancheng-zhi Thanks for your great work! Could you please disclose the training time and GPU cost? Thanks.

**Comment by zeyuanyin** [comment]
> Sorry there is no plan to release training code. However, the loss function is same as Wan2.1 itself, a standard flow matching loss.

Hi @tiancheng-zhi, thanks for your work. I am wondering if you built your training code based on https://github.com/modelscope/DiffSynth-Studio/tree/main/examples/wanvideo/model_training
